serve/mlc_serve/api/handler.py

```diff
@@ -229,15 +237,32 @@ async def collect_result_stream(
             if seq.is_finished:
                 assert seq.finish_reason is not None
                 finish_reasons[seq.index] = seq.finish_reason.value  # type: ignore
-
-    choices = [
-        ChatCompletionResponseChoice(
+    
+    choices = []
+    for index, (chunks, finish_reason) in enumerate(zip(sequences, finish_reasons)):
+        content = []
+        if logprob_infos[index] != []:
+            for logprob_info in logprob_infos[index]:
+                top_logprobs = [TopLogprobs(
+                    token=str(token),
+                    logprob=float(logprob),
+                    # TODO(vvchernov): implement bytes bases on https://platform.openai.com/docs/api-reference/chat/object
+                    bytes=None,
+                ) for token, logprob in logprob_info[1]]
+                content.append(LogprobsContent(
+                    token=str(logprob_info[0][0]),
+                    logprob=float(logprob_info[0][1]),
```

Comment:
Please make it obvious what the indexing by 0 and 1 here means.

Reply:
Ditto.